test/e2e_node/README.md

```diff
@@ -0,0 +1,109 @@
+# Node End-To-End tests
+
+Node e2e tests start kubelet and minimal supporting infrastructure to validate the kubelet on a host.
+Tests can be run either locally, against a remote host or against a GCE image.
+
+*Note: Linux only. Mac and Windows unsupported.*
+
+## Running tests locally
+
+etcd must be installed and on the PATH to run the node e2e tests.  To verify etcd is installed: `which etcd`.
+You can find instructions for installing etcd (here)[https://github.com/coreos/etcd/releases].
+
+Run the tests locally: `make test_e2e_node`
+
+Running the node e2e tests locally will build the kubernetes go source files and then start the
+kubelet, kube-apiserver, and etc binaries on localhost before executing the ginkgo tests under
+test/e2e_node against the local instance.
+
+## Running tests against a remote host
+
+The node e2e tests can be run against one or more remote hosts using one of
+* `test/e2e_node/jenkins/e2e-node-jenkins.sh` (gce only)
+* `test/e2e_node/runner/run_e2e.go` (requires ssh)
+* `./e2e_node.test` (requires host access w/ sudo)
+
+*Requires password-less sudo access (over ssh) on the remote host to run the kubelet and kube-apiserver.*
+
+### Configuring a new remote host for testing
+
+A minimal environment is required before tests can be run on a remote host.  Most of this can be setup for you using:
+
+`curl -L https://goo.gl/kxDoYY | sudo bash`
```

Comment:
We should not tell users to do this. Piping any script to a root shell is a bad idea (we should explicitly sudo the necessary elements - which it looks like the script already does). Also, piping a shortlink is bad practice (it obfuscates what you're actually sending).

I think you should check in the gist this points to, and then curl from the k8s repository which will have a lot more trust associated with it.